kafka_quota_enforcer/src/main/java/com/tesla/data/quota/enforcer/QuotaEnforceCommand.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2021. Tesla Motors, Inc. All rights reserved.
+ */
+
+package com.tesla.data.quota.enforcer;
+
+import com.tesla.data.enforcer.EnforceCommand;
+import com.tesla.data.enforcer.Enforcer;
+
+import kafka.zk.AdminZkClient;
+import kafka.zk.KafkaZkClient;
+import org.apache.kafka.common.security.JaasUtils;
+import org.apache.kafka.common.utils.Time;
+
+import java.util.Map;
+
+
+public class QuotaEnforceCommand extends EnforceCommand<ConfiguredQuota> {
+  private KafkaZkClient zkClient;
+  public QuotaEnforceCommand(){
+    // DO NOT REMOVE, this is needed by jcommander
+  }
+
+  @Override
+  protected Enforcer<ConfiguredQuota> initEnforcer() {
+    Map<String, Object> zookeeperConfig = zookeeperConfig();
+    this.zkClient = KafkaZkClient.apply(
+        zookeeperConfig.get("connect").toString(),
+        JaasUtils.isZkSecurityEnabled(),
+        Integer.parseInt(zookeeperConfig.get("sessionTimeoutMs").toString()),
+        Integer.parseInt(zookeeperConfig.get("connectionTimeoutMs").toString()),
+        Integer.MAX_VALUE,
+        Time.SYSTEM,
+        "kafka.server", // default used in the client
+        "SessionExpireListener" // default used in the client
+    );
```

Comment:
A general comment, its a good practice to declare the variable as final as much as possible in many cases it doesn't have an obvious benefit, but in multi-threaded code, it can protect you from bugs